src/CattyUITests/ScriptCollectionView/ListTests.swift

```diff
@@ -71,56 +71,6 @@ class ListTests: XCTestCase {
     }
 
     func testCreateAndSelectList() {
-        let listName = "TestList"
```

Comment:
Is there a special reason why you removed those existing tests? Are they not used anymore?

Reply:
Yes, they are not used anymore. Because we don't have a "choose" button anymore.